pages/Copper-Dust.md

```diff
@@ -0,0 +1,19 @@
+Copper Dust is one of the nine [dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts) added in Slimefun.
+
+## Obtaining
+
+Copper Dust has a ~11.11% chance of being extracted from [Sifted Ore](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Sifted-Ore), by processing it through an [Ore Washer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ore-Washer) or an [Electric Dust Washer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Dust-Washer).
+
+## Usage
+Copper Dust can be smelted into [Copper Ingots](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Copper-Ingot) in a [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or an [Electric Ingot Factory](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Factory).
+
+It is also used in the following [alloys](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ingots#Alloys) for the [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or the [Electric Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Smeltery):
+* [Bronze Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Bronze-Ingot)
+* [Duralumin Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Duralumin-Ingot)
+* [Billon Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Billon-Ingot)
+* [Brass Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Brass-Ingot)
+* [Corinthian Bronze Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Corinthian-Bronze-Ingot)
+* [Nickel Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Nickel-Ingot)
+* [Cobalt Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Cobalt-Ingot)
+
+Copper is one of the most used dusts in SlimeFun recipes, either as dust or as [Copper Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Copper-Ingot).
```

Comment:
:eyes:

Reply:
Yeah, this is personal observation, based on the usage statistics I made. Happy to remove these and maybe create a separate page for more statistical analysis?

Reply:
Fixed "Slimefun" issues